scripts/common.py

```diff
@@ -422,6 +428,26 @@ def check_prs_for_current_release_are_released(repo):
                 'released before release summary generation.')
 
 
+def kill_processes_based_on_regex(pattern):
+    """Kill processes based on provided regex.
```

Comment:
Kill any processes whose names (?) match the provided regex.

(Please replace "names" with whatever is appropriate.)

Reply:
Done